app/jobs/cleanup_job.rb

```diff
@@ -0,0 +1,7 @@
+class CleanupJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    Target.where('created_at <= ?', 1.week.ago).destroy_all
```

Comment:
As a suggestion, this kind of things can be done in a scope. See https://guides.rubyonrails.org/active_record_querying.html#scopes

Reply:
Done! Let me know if like this its ok, or there is something to improve. Thanks!

Reply:
I think you forgot to commit the changes 🤔

Reply:
Mb now should appear the changes.